media/open_zim.ksy

```diff
@@ -0,0 +1,356 @@
+  cluster_ptr:
+    seq:
+      - id: ptr
+        type: u8
+    instances:
+      cluster:
+        pos: ptr
+        type: cluster(ptr)
+    types:
+      cluster:
+        # BUG: JUNK. Seems to be positioned to a wrong offsets. info.reserved is not 0
+        params:
+          - id: ptr
+            type: u8
+        seq:
+          - id: info
+            type: info_t
+          - id: blobs_ptrs_compressed
+            process: kaitai.compress.lzma(2)
+            size: info.ptr_size * blob_count_plus_one
+            type: blobs_ptrs(blob_count)
+            if: info.compression == compression::lzma2
+          - id: blobs_ptrs_uncompressed
+            type: blobs_ptrs(blob_count)
+            if: info.compression == compression::none or info.compression == compression::default
+        instances:
+          first_blob_offset:
+            value: ptr + 1 # info_t.size
+          blobs: 
+            type: blob(_index)
+            repeat: expr
+            repeat-expr: blob_count
+          first_blob_ptr_compressed:
+            pos: first_blob_offset
+            process: kaitai.compress.lzma(2)
+            size: 8 + _root.minimal_xz_lzma_size # max(8, 4) + minimal_xz_lzma_size
+            type: blob_ptr_u
+            if: info.compression == compression::lzma2
+          first_blob_ptr_uncompressed:
+            pos: first_blob_offset
+            type: blob_ptr_u
+            if: info.compression == compression::none or info.compression == compression::default
+          first_blob_ptr_u_v:
+            value: (info.compression == compression::lzma2?first_blob_ptr_compressed:first_blob_ptr_uncompressed).as<blob_ptr_u>.value
+          blob_count_plus_one:
+            value: first_blob_ptr_u_v / info.ptr_size
+          blob_count:
+            value: blob_count_plus_one - 1
+          blobs_ptrs:
+            value: (info.compression == compression::lzma2?blobs_ptrs_compressed:blobs_ptrs_uncompressed).ptrs
+        types:
+          blobs_ptrs:
+            params:
+              - id: count
+                type: u8
+            seq:
+              - id: ptrs
+                type: blob_ptr_u
+                repeat: eos
+            instances:
+              info:
+                value: _parent.info
+          blob:
+            params:
+              - id: idx
+                type: u8
+            instances:
+              data:
+                pos: _parent.ptr + _parent.blobs_ptrs[idx].as<blob_ptr_u>.value
+                size: _parent.blobs_ptrs[idx+1].as<blob_ptr_u>.value - _parent.blobs_ptrs[idx].as<blob_ptr_u>.value
+          info_t:
+            #seq:
+            #  - id: compression
+            #    type: b4
+            #    enum: compression
+            #  - id: ptr_is_u8
+            #    -orig-id: extended
+            #    type: b1
+            #  - id: reserved
+            #    type: b3
+            seq:
+              - id: reserved
+                type: b3
+              - id: ptr_is_u8
```

Comment:
I'd rather pick `is_ptr_u8`, because starting with `is_` prefix is customary for boolean getters (accessors) in programming languages with classes, and also the question sounds better as "Is `ptr` `u8`?".